integration_test/workloads_test.go

```diff
@@ -24,6 +24,7 @@ func (s *connectedSuite) successfulIntercept(tp, wl, port string) {
 		2*time.Second, // polling interval
 	)
 
+	itest.TelepresenceOk(ctx, "loglevel", "trace")
```

Comment:
It'd be good to restore to the previous state
```suggestion
	itest.TelepresenceOk(ctx, "loglevel", "trace")
	defer itest.TelepresenceOk(ctx, "loglevel", "debug")
```